### ISSUE TITLE
CLI glob problem for recursively targeting files 

### ISSUE THREAD

**Issue by akiva** [imagemin/imagemin#87]
Given a structure:

```
./source/images/
./source/images/test/
./source/images/foo/
```

where each directory contains images, I am unable to run any successful imagemin task where I glob this path and wish to have the output respect the input structure.

If I issue the following command: `imagemin ./source/images/**/* ./build/images`, `./build/images` will only contain images from the subdirectories, ignoring those in `./source/images` directly. If I choose my glob with file extensions (`./source/images/**/*.{png,jpg,svg}`, etc.), all images are thrown into  `./build/images`, however without the subdirectories as desired (naturally due to the extension specifier). I have tried a few variations here, such as targeting the entire source image directory (`./sources/images`) to no avail.

**Comment by zanona** [comment]
Agree, it would be great to have `imagemin -r dir/** build` to recursively _copy_ the exact structure as the original directory but with images compressed.

**Comment by kevva** [comment]
I though that worked already. Will have a look at this later today.

**Comment by zanona** [comment]
Perhaps this can be merged into #78 since apparently that still an issue with the current version?

**Comment by sktt** [comment]
Hey @kevva! I (re-?)added recursive support in imagemin-cli. Would be happy if you have comments on it :-)

As I've mentioned, using globbing pattern would indeed select all files, but won't keep the folder structure which I believe most of us want.

**Comment by sanfilippopablo** [comment]
May I ask why isn't this solved yet?

**Comment by filoxo** [comment]
Yet another bump on this. Looks like there are 2 submitted PRs (#196, #225) that are attempting to solve this; both are 6+ months old. Is there anything else the community can do to get this feature pushed though? Thanks!

**Comment by alexbudin** [comment]
Has this been fixed?

**Comment by freereikicourse** [comment]
Yeah same problem here.. Not sure how is supposed to work if is not doing this critical feature..

**Comment by cigar9** [comment]
@sktt 's PR[#11](https://github.com/imagemin/imagemin-cli/pull/11) works fine.  Not merged yet?

**Comment by Dobby89** [comment]
This definitely needs fixing soon. It sounds like there's been a possible fix around for a year!

**Comment by chinjon** [comment]
I too think this would be useful, looks like there are two PRs - but they're failing. Any plans to address these?
